lib/endpoints/class-wp-rest-terms-controller.php

```diff
@@ -138,7 +138,7 @@ public function get_items( $request ) {
 	public function get_item( $request ) {
 
 		$term = get_term_by( 'term_taxonomy_id', (int) $request['id'], $this->taxonomy );
-		if ( ! $term ) {
+		if ( ! $term || $term->taxonomy !== $this->taxonomy ) {
```

Comment:
> How is this possible? My understanding is `get_term_by` will return null for a `term_id` that doesn't belong to that taxonomy, and no shared terms should now have stopped that being unexpected.

This changed in 4.4, see #1644 and [WP30620](https://core.trac.wordpress.org/ticket/30620).